wgpu-hal/src/gles/adapter.rs

```diff
@@ -289,6 +289,9 @@ impl super::Adapter {
             wgt::DownlevelFlags::ANISOTROPIC_FILTERING,
             extensions.contains("EXT_texture_filter_anisotropic"),
         );
+        if cfg!(not(target_arch = "wasm32")) {
+            downlevel_flags.set(wgt::DownlevelFlags::READ_ONLY_DEPTH_STENCIL, true);
+        }
```

Comment:
This isn't true, iirc according to the gl spec RODS is always UB. It happens to work because it's not a real race condition but I don't think you can technically do it.